src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -24,7 +15,42 @@ interface ItemProps {
 
 const Item: React.FC<ItemProps> = props => {
   const { item } = props
-  if (item.__typename === "Artwork") {
+  const itemType = item.__typename
+
+  const getImage = item => {
+    if (itemType === "Artwork") {
+      return item.image?.url
+    } else if (itemType === "Show") {
+      return item.coverImage?.url
+    } else {
+      return null
+    }
+  }
+
+  const itemDetails = item => {
+    if (item.__typename === "Artwork") {
+      return [
+        <Sans size="4" weight="medium" color="white100">
+          {item.artistNames}
+        </Sans>,
+        <Sans size="2" color="white100">
+          {item.title} / {item.date}
+        </Sans>,
+      ]
+    } else if (item.__typename === "Show") {
+      const itemLocation = item.fair?.location?.city
+      return [
+        <Sans size="4" weight="medium" color="white100">
+          {item.fair.name}
+        </Sans>,
+        <Sans size="2" color="white100">
+          {itemLocation && `${itemLocation}, `} {item?.fair?.exhibitionPeriod}
```

Comment:
I'm pretty `exhibitionPeriod` should always be present because I believe `start_at` and `end_at` are required fields for a show and that's what `exhibitionPeriod` is derived from. @starsirius do you know for sure?

Do y'all think this needs additional guards?

Reply:
I think the guards here are sufficient unless more details need to be added.

Reply:
I think it's safe to assume we will have `start_at` and `end_at` for a [fair](https://github.com/artsy/gravity/blob/fa51a3b44c1876101e664954fd76278fae6358d9/app/models/domain/fair.rb#L131).